RevenueCatUI/CustomerCenter/Data/CustomerCenterConfigData+Mock.swift

```diff
@@ -135,7 +135,8 @@ extension CustomerCenterConfigData {
     }
 
     @available(iOS 14.0, *)
-    static let `default` = mock()
+    // swiftlint:disable missing_docs
+    @_spi(Internal) public static let `default` = mock()
```

Comment:
Why is this one needed? It's part of RevenueCatUI, so...

Reply:
You're right!! 🙌 
The compiler was failing to find it, but looks like it was just a hiccup. Removing it